tool/gravity/cli/rpcagent.go

```diff
@@ -130,30 +130,40 @@ var agentFunctions map[string]agentFunc = map[string]agentFunc{
 	constants.RPCAgentSyncPlanFunction: executeSyncOperationPlan,
 }
 
-func rpcAgentDeploy(localEnv, updateEnv *localenv.LocalEnvironment, leaderParams, nodeParams string) error {
-	clusterEnv, err := localEnv.NewClusterEnvironment()
+func rpcAgentDeploy(localEnv *localenv.LocalEnvironment, leaderParams, nodeParams string) error {
+	_, err := rpcAgentDeployHelper(context.Background(), localEnv, leaderParams, nodeParams)
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	return nil
+}
+
+func rpcAgentDeployHelper(ctx context.Context, localEnv *localenv.LocalEnvironment, leaderParams, nodeParams string) (credentials.TransportCredentials, error) {
```

Comment:
Couldn't really come up with a better name for this. There is already `deployAgents` method below :)